bin/disco_elasticache.py

```diff
@@ -4,12 +4,14 @@
 
 Usage:
     disco_elasticache.py [--debug] [--env ENV] list
-    disco_elasticache.py [--debug] [--env ENV] update [--cluster CLUSTER]
+    disco_elasticache.py [--debug] [--env ENV] listsnapshots [--cluster CLUSTER]
```

Comment:
Oh right. I was thinking of `disco_route53.py list-records`. But i also just discovered `disco_rds.py cleanup_snapshots` and `disco_alarms.py update_metrics`. We are just not consistent anywhere...